node/src/reactor/main_reactor/reactor_state.rs

```diff
@@ -4,9 +4,58 @@ use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
 /// The state of the reactor.
+#[cfg_attr(doc, aquamarine::aquamarine)]
+/// ```mermaid
+/// flowchart TD
+///     style Start fill:#66ccff,stroke:#333,stroke-width:4px
+///     style End fill:#66ccff,stroke:#333,stroke-width:4px
+///     
+///     Start --> Initialize
+///     Initialize --> CatchUp
+///     CatchUp --> KeepUp
+///     KeepUp --> CatchUp
+///     KeepUp --> Validate
+///     Validate --> KeepUp
+///     CatchUp --> ShutdownForUpgrade
+///     KeepUp --> ShutdownForUpgrade
+///     Validate --> ShutdownForUpgrade
+///     CatchUp --> Upgrading
+///     CatchUp -->|at genesis| Validate
+///     Upgrading --> CatchUp
+///     ShutdownForUpgrade --> End
+/// ```
+/// ```mermaid
+/// flowchart TD
+///     style Start fill:#66ccff,stroke:#333,stroke-width:4px
+///     style End fill:#66ccff,stroke:#333,stroke-width:4px
+///     style F fill:#ffcc66,stroke:#333,stroke-width:4px
+///     style G fill:#ffcc66,stroke:#333,stroke-width:4px
+///     title[CatchUP process]
```

Comment:
```suggestion
///     title[CatchUp process]
```

Reply:
Updated in https://github.com/casper-network/casper-node/commit/09ca60ebb01c4151d3d19c75ec6b9715ca663d81